code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -253,23 +253,28 @@
 /obj/item/modular_computer/pda/nukeops
 	name = "nuclear pda"
 	device_theme = "syndicate"
-	has_pda_programs = FALSE
+	has_pda_programs = TRUE
 	comp_light_luminosity = 6.3 //matching a flashlight
 	light_color = COLOR_RED
 	greyscale_config = /datum/greyscale_config/tablet/stripe_thick
 	greyscale_colors = "#a80001#5C070F#000000"
-
+	long_ranged = TRUE
 	starting_programs = list(
-		/datum/computer_file/program/ntnetdownload/syndicate,
 		/datum/computer_file/program/radar/fission360,
 	)
 
-/obj/item/modular_computer/pda/nukeops/emag_act(mob/user)
-	if(!enabled)
-		to_chat(user, span_warning("You'd need to turn the [src] on first."))
-		return FALSE
-	to_chat(user, span_notice("You swipe \the [src]. It's screen briefly shows a message reading \"MEMORY CODE INJECTION DETECTED AND SUCCESSFULLY QUARANTINED\"."))
-	return FALSE
+/obj/item/modular_computer/pda/nukeops/Initialize(mapload)
+	. = ..()
+	obj_flags |= EMAGGED //Mostly for consistency purposes; the programs will do their own emag handling
+	var/newemag = FALSE
+	for(var/datum/computer_file/program/app in stored_files)
+		if(!istype(app))
+			continue
+		if(app.run_emag())
+			newemag = TRUE
```

Comment:
Why're we doing this instead of just calling ``emag_act``?

Reply:
Because the PDA doesn't start on, and PDA emag_act fails if the PDA isn't on.

Reply:
maybe add an arg ``forced`` or ``silent`` to it?

Reply:
From `code\modules\modular_computers\computers\item\computer.dm`
```
/obj/item/modular_computer/emag_act(mob/user)
	if(!enabled)
		to_chat(user, span_warning("You'd need to turn the [src] on first."))
		return FALSE
```

Reply:
Add an arg that will bypass it is what I meant

Reply:
> maybe add an arg `forced` or `silent` to it?

To the base emag_act?

Reply:
> Add an arg that will bypass it is what I meant

yeah i can try that